### PR TITLE
Updates (15/08/2018)

### PR DESCRIPTION
**Pull request by DaniellMesquita**

### CODE REVIEW

File: 3. cloning zites.md [hunk 1]
```diff
@@ -36,17 +36,21 @@ One more word about trackers. You need a static IP or Tor connection to share zi
 
 Anybody can connect to your blog while there is a running computer (serving your zite). But you probably want to shutdown it at night. So it's a good idea to have a computer always working.
 
-Fortunately, there are such computers: *ZeroNet proxies*. [bit.surf](http://bit.surf:43110/) is one of them. You should use them as you use local ZeroNet. On proxies, *ZeroTalk* address is `http://bit.surf:43110/`
+Fortunately, there are such computers: *ZeroNet proxies*. [bit.surf](http://bit.surf:43110/) is one of them. You should use them as you use local ZeroNet. On proxies, *ZeroTalk* address is `http://bit.surf:43110/1TaLkFrMwvbNsooF4ioKAY9EuxTBTjipT`
 
 When your blog is ready, visit `http://bit.surf:43110/{zite public key}/`. You can get public key from address bar: `http://127.0.0.1:43110/{zite public key}/`.
 
+*Note*: bit.surf is no longer working.
```
Comment (imachug) [inline 3. cloning zites.md]:
```suggestion
*Note*: bit.surf is no longer working. Use [0net.io](https://0net.io) and [@amorgan's proxy](http://zn.amorgan.xyz) instead.
```

File: 4. creating static zite.md [hunk 2]
```diff
@@ -185,7 +185,7 @@ Scroll to the bottom and change `Site title` and `Site description` parameters.
 
 Now let's publish your zite. First, press `Sign` button in the very bottom of the sidebar. Your `content.json` now holds updated information about files and is signed correctly. Now press `Publish`. You will probably see `Content publish failed` message. That's OK, as no other peers have downloaded your zite.
 
-Now, if you have a static IP or Tor mode is on, your zite can be accessed from other peers while your computer is on. If you want to turn your computer down, visit `bit.surf`: `http://bit.surf:43110/{zite public key}/` (You can get public key from address bar of from sidebar).
+Now, if you have a static IP or Tor mode is on, your zite can be accessed from other peers while your computer is on. If you want to turn your computer down, visit `ZeroGate`: `http://zerogate.tk/{zite public key}/` (You can get public key from address bar of from sidebar).
```
Comment (imachug) [inline 4. creating static zite.md]:
```suggestion
Now, if you have a static IP or Tor mode is on, your zite can be accessed from other peers while your computer is on. If you want to turn your computer down, visit `0net.io`: `http://0net.io/{zite public key}/` (You can get public key from address bar of from sidebar).
```

File: 13. ugly things.md [hunk 3]
```diff
@@ -67,6 +67,42 @@ From ZeroMe source code:
...
+    "permission_rules": {
+        ".*": {
+            "files_allowed": "data.json",
+            "max_size": 50000,
+            "signers": ["1Abc"]
+        }
+    },
+    ...
+
+We add `signers` to `permission_rules` items! Now, `1Abc` can sign all users' content. And if you move `signers` to a new item, `.*bot@.*`, `1Abc` will be able to sign only `...bot` user content.
```
Comment (imachug) [inline 13. ugly things.md]: Didn't think about this before! Good idea.

File: 3. cloning zites.md (hunk 1 above)
Comment (DaniellMesquita) [reply 3. cloning zites.md]: I'll edit about bit.surf after this PR gets merged

File: 13. ugly things.md (hunk 3 above)
Comment (DaniellMesquita) [reply 13. ugly things.md]: Well, isn't you that wrote it?
It was already come when I downloaded it

Comment (DaniellMesquita) [reply 13. ugly things.md]: @krixano

Comment (imachug) [reply 13. ugly things.md]: Wait, what?! Yeah, maybe I added it... Yeah, I think so.

**Comment by DaniellMesquita** [comment]
@imachug 
Your revisions (and some additions) are done on those commits:
https://github.com/imachug/The-Truth-about-ZeroNet/commit/c26b83e86914517324aa212060c822254968143f, https://github.com/imachug/The-Truth-about-ZeroNet/commit/d262ee0aa9b0cb076790363effa0bbc6935bfae2, https://github.com/imachug/The-Truth-about-ZeroNet/commit/bbaa1f3df3ffa15da80874d6907b77df62f028c4

**Pull request merged**